.github/workflows/tests.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
   run-tests:
     strategy:
       matrix:
-        os: [macos-10.15, ubuntu-18.04, ubuntu-20.04]
```

Comment:
I suspect Windows would need its own entry or bash installed or make the steps conditional

Reply:
Already done :D